src/coordinator/roles.js

```diff
@@ -411,7 +388,7 @@ class Roles {
     return this._request.delete(
       `${
         this._baseUrl
-      }/projects/${projectSlug}/environments/${projectEnvironmentSlug}/roles/${roleId}`
+      }/roles/${roleId}/project_environments/${projectEnvironmentSlug}`
```

Comment:
So these two are the duplicate routes you were talking about - given that the old one was roles on an environment, should we move forward with the route for `/:environment_slug/roles/:role_id` rather than the reverse?

Reply:
There is now just one way to add a role to an envrionment in contxt-coordinator at this new URL. Change can be found here: https://github.com/ndustrialio/contxt-coordinator/pull/306